include/abb_librws/v2_0/rws_client.h

```diff
@@ -362,6 +362,9 @@ class RWSClient
    */
   static std::string generateFilePath(const FileResource& resource);
 
+  static bool shouldRetry(const POCOResult result,
+                          std::set<Poco::Net::HTTPResponse::HTTPStatus> const& retry_required_statuses);
+
```

Comment:
I think it is over-complicated with an extra function and an `std::set`.
```suggestion
  static bool shouldRetry(Poco::Net::HTTPResponse::HTTPStatus status);

```